packages/navitem/src/react/index.js

```diff
@@ -2,22 +2,14 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 import { alignments } from '../vars/index.js'
-import { Bar, Button, Container } from './common.js'
 import { HorzLayout } from './horz.js'
 import { VertLayout } from './vert.js'
 
 const NavItem = React.forwardRef((props, forwardedRef) => {
-  return (
-    <Container>
-      <Button {...props} ref={forwardedRef}>
-        {props.alignment === 'vertical' ? (
-          <VertLayout {...props} />
-        ) : (
-          <HorzLayout {...props} />
-        )}
-      </Button>
-      <Bar {...props} />
-    </Container>
+  return props.alignment === 'vertical' ? (
+    <VertLayout ref={forwardedRef} {...props} />
```

Comment:
the layout component props don't support the `alignment` prop but we're passing it...maybe we shouldn't

Reply:
@danethurber ... I'm just passing everything with the spread operator.  It's an internal component.  Do you think we need to pass each prop explicitly instead?

Reply:
react will still throw a warning because `alignment` isn't a dom attrs. i'd just do a some rest args and spread

```
const {alignment, ...rest} = props

<Layout {...rest} />
```

you know the dril

Reply:
React seems happy.  `npm run lint` has no errors and Travis CI is agreeing.  Does that change your mind?

Reply:
well i trust you...so now i'm doubting myself. one sec

Reply:
looks like the `Button` is filtering props and so alignment is not added to the dom because of other prop filtering. that seems like a side-effect and we should be more explicit - thoughts

Reply:
Sure, I'll be explicit.

Just contrasting that explicitness principle in my mind to the potential use cases for spreading props: If we can't spread to internal components which we fully control because of a lack of explicitness, I'm not sure where spreading is appropriate.

Prepping a commit...

Reply:
yeah. it's also kinda of an edge case and really just a react warnings. not sure the best course of action but talking through it will continue to frame the issue